sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -322,15 +326,16 @@ def get_subscription_receiver(self, topic_name, subscription_name, **kwargs):
          sessionful subscription, otherwise it must be None. In order to receive messages from the next available
          session, set this to ~azure.servicebus.NEXT_AVAILABLE_SESSION.
         :paramtype session_id: Union[str, ~azure.servicebus.NEXT_AVAILABLE_SESSION]
-        :keyword Optional[SubQueue] sub_queue: If specified, the subqueue this receiver will connect to.
+        :keyword Optional[Union[ServiceBusSubQueue, str]] sub_queue: If specified, the subqueue this receiver will
+         connect to.
```

Comment:
Do we want to add to the docstring something like: Possible values include 'deadletter', 'transferdeadletter'.

Reply:
Mmm, do you worry that clutters it vs. someone being able to eyeball the enum?  Adds maintenance overhead too to keep those in parity.  (I hate the fact that I have to make this counterargument, if there was _any_ sort of docstring formatting I'd feel better adding more content to it.  If you feel like this'd be a big help or we have precedent let me know and I'll simply align.

Reply:
I used sqlalchemy as my tiebreaker.  It lists enum parameters, e.g. ISOLATION_LEVEL.  Add it is, solid call++